py/selenium/webdriver/remote/switch_to.py

```diff
@@ -87,7 +87,7 @@ def frame(self, frame_reference: Union[str, int, WebElement]) -> None:
                 try:
                     frame_reference = self._driver.find_element(By.NAME, frame_reference)
                 except NoSuchElementException as exc:
-                    raise NoSuchFrameException(frame_reference) from exc
+                    raise NoSuchFrameException(str(frame_reference)) from exc
```

Comment:
I don't think this is needed. This already gets implicitly converted to string.

Reply:
this resolved the mypy error .